File: Week 01/id_701/leetcode_283_701.cs
public class Solution
{
    public void MoveZeroes(int[] nums)
    {
        int id = 0;
        for (int i = 0; i < nums.Length; i++)
        {
            if (nums[i] != 0)
            {
                int temp = nums[id];
                nums[id] = nums[i];
                nums[i] = temp;
                id += 1;
            }
        }
    }
}

File: Week 01/id_701/leetcode_283_701.py
class Solution:
    def moveZeroes(self, nums: list) -> None:
        for i in range(len(nums)):
            if nums[i] == 0:
                nums.remove(0)
                nums.append(0)

    def moveZeroes(self, nums: list) -> None:
        id = 0
        for i in range(len(nums)):
            if nums[i] != 0:
                nums[id], nums[i] = nums[i], nums[id]
                id += 1
